packages/core/src/components/viewer-grid/viewer-grid.tsx

```diff
@@ -95,11 +97,13 @@ export class ViewerGrid {
   }
 
   shrinkMain = () => {
-    this.mainSize = this.mainSize == "large" ? "medium" : "small";
+    this.mainSize = this.mainSize === 'large' ? 'medium' : 'small';
+    this.srSize = this.srSize === 'breed' ? 'middel' : 'smal';
```

Comment:
Het is onnodig om twee keer de huidige staat bij te houden. Je kan in de `render()` prima de `mainSize` resolven tot een leesbaar label.

Je kan een `private getSizeLabel(): string` maken, of een `private sizeLabelMap: { [size: MainSize]: string };` met `{this.sizeLabelMap[this.mainSize]}`.